src/benchmarks/micro/corefx/System.Text.Json/Serializer/ReadJson.Dictionary.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public class ReadDictionary
+    {
+        private const string _jsonString = @"{""Hello"":""World"",""Hello2"":""World2""}";
```

Comment:
It would be useful to see perf of JSON with a lot more key/value pairs as well (say 100).

Reply:
I would also strongly recommend a bigger dictionary.

If we try to serialize a very small one then most probably most of the time will be spend not in the serializaiton of the dictionary but elsewhere: in searching for the type information, properies metadata etc.

BTW you can use https://github.com/dotnet/performance/blob/2e05afce5332d5e0a308e6208112b4205d3faf75/src/harness/BenchmarkDotNet.Extensions/ValuesGenerator.cs#L27 to get unique strings and populate the dictionary:

```cs
Dictionary<string, string> dictionary = ValuesGenerator.ArrayOfUniqueValues<string>(100).ToDictionary(value => value);
```